doc/external_cpp.rst

```diff
@@ -131,9 +131,10 @@ is set to ``True``.
                                   verbose=True,
                                   allow_undefined=True,
                                   includes=include_files,
-                                  include_dirs=[include_dir],
+                                  include_dirs=include_dir,
```

Comment:
Let's change the name of the variable too. `include_dirs` is a better variable name than `include_dir` given that it is a list.

Reply:
Yes, that sounds reasonable